### PR TITLE
Updating install scripts

### PR DESCRIPTION
**Pull request by pranavkm**
Update to https://github.com/aspnet/KoreBuild/pull/44

### CODE REVIEW

File: build/KoreBuild.ps1 [hunk 1]
```diff
@@ -13,6 +13,7 @@ $koreBuildFolder = $koreBuildFolder.Replace($repoFolder, "").TrimStart("\")
 $dotnetVersionFile = $koreBuildFolder + "\cli.version.win"
 $dotnetChannel = "beta"
 $dotnetVersion = Get-Content $dotnetVersionFile
+$dotnetCLINew = $env:KOREBUILD_DOTNET_CLI_NEW
```
Comment (pranavkm) [inline build/KoreBuild.ps1:16]: We'll try and keep compat with the old script until the gate succeeds.

File: build/KoreBuild.ps1 [hunk 2]
```diff
@@ -25,16 +26,23 @@ if ($env:KOREBUILD_DOTNET_VERSION)
...
 $dotnetLocalInstallFolder = "$env:LOCALAPPDATA\Microsoft\dotnet\cli"
 $dotnetLocalInstallFolderBin = "$dotnetLocalInstallFolder\bin"
-$newPath = "$dotnetLocalInstallFolderBin;$env:PATH"
+$newPath = "$dotnetLocalInstallFolder;$dotnetLocalInstallFolderBin;$env:PATH"
 if ($env:KOREBUILD_SKIP_RUNTIME_INSTALL -eq "1") 
 {
     Write-Host "Skipping runtime installation because KOREBUILD_SKIP_RUNTIME_INSTALL = 1"
     # Add to the _end_ of the path in case preferred .NET CLI is not in the default location.
-    $newPath = "$env:PATH;$dotnetLocalInstallFolderBin"
+    $newPath = "$env:PATH;$dotnetLocalInstallFolder;$dotnetLocalInstallFolderBin"
```
Comment (pranavkm) [inline build/KoreBuild.ps1:34]: Will clean this up once we have transitioned to the new scripts.

File: build/dotnet/install-old.ps1 [hunk 3]
```diff
@@ -0,0 +1,108 @@
+#
```
Comment (pranavkm) [inline build/dotnet/install-old.ps1:1]: The two old scripts are the result of `git mv install.ps1 install-old.ps1`. Will get removed once we've updated

**Comment by pranavkm** [comment]
cc @NTaylorMullen \ @davidfowl \ @victorhurdugaci

File: build/KoreBuild.ps1 (hunk 1 above)
Comment (victorhurdugaci) [inline build/KoreBuild.ps1:16]: since it's a flag, name it `useDotnetCliNew`

**Comment by victorhurdugaci** [comment]
:shipit:

**Comment by NTaylorMullen** [comment]
:shipit:

**Pull request merged**